Gopkg.lock

```diff
@@ -2,79 +2,87 @@
 
 
 [[projects]]
-  name = "bitbucket.org/ww/goautoneg"
-  packages = ["."]
-  revision = "'75cd24fc2f2c2a2088577d12123ddee5f54e0675'"
-
-[[projects]]
+  branch = "master"
```

Comment:
But this also upgrades `github.com/NYTimes/gziphandler` to master

Reply:
dep did that not me, probably because those projects don't have releases or any other branches.

Reply:
I'm guessing you ran a `dep ensure -update` without specifying a package, so it attempted to upgrade all packages (not something I'm sure we want to do)